docs/ref/settings.txt

```diff
@@ -1675,6 +1675,21 @@ renderers are:
 * ``'``:class:`django.forms.renderers.Jinja2`\ ``'``
 * ``'``:class:`django.forms.renderers.TemplatesSetting`\ ``'``
 
+.. setting:: EARLY_ADOPT_URLFIELD_ASSUME_HTTPS
+
+``EARLY_ADOPT_URLFIELD_ASSUME_HTTPS``
+-------------------------------------
+
+.. versionadded:: 5.0
+.. deprecated:: 5.0
+
+Default: ``False``
+
+It's a transitional setting helpful in early adoption of ``"https"`` as the new
+default value of :attr:`URLField.assume_scheme
+<django.forms.URLField.assume_scheme>`. Set this to ``True`` to assume
+``"https"`` in ``URLField`` during the Django 5.x release cycle.
```

Comment:
“It’s a” is unnecessary

“helpful” is vague

`URLField` -> `forms.URLField` clarifies where this applies

I think we can merge the info into one sentence:

```suggestion
Set this transitional setting to ``True`` to opt into using ``"https"`` as the
default value of
:attr:`forms.URLField.assume_scheme <django.forms.URLField.assume_scheme>`
during the Django 5.x release cycle.
```